modules/attributes/pages/document-attributes-reference.adoc

```diff
@@ -39,7 +39,7 @@ When you set an attribute in the body, it's visible from the point it's set unti
 
 == Compliance attributes
 
-[%autowidth,cols="m,^,,^,^,"]
+[%autowidth,cols="~m,^~,~,^~,^~,~"]
```

Comment:
`^~,~` is now my favorite smiley :)

Reply:
Just to note, this is a forward-looking change. Right now `~` is assumed, but if we ever want to be able to have relative widths for an autowidth table, we would need to be explicit about it.